src/lib/hooks/useNotificationHandler.ts

```diff
@@ -231,6 +242,8 @@ export function useNotificationsHandler() {
           const payload = e.notification.request.trigger
             .payload as NotificationPayload
```

Comment:
This typecast was ignoring the possibility of `undefined` as specified in the types for `notification.request.trigger.payload`